APPSEC-1088: S6385 Improve RSPEC (#3092)
SonarSource/rspec: rules/S6385/description.adoc

Defining a custom role for a Subscription or a Management group that allows all actions will give them the same capabilities as the built-in Owner role.

> It's recommended to limit the number of subscription owners in order to mitigate the risk of being breached by a compromised owner.

Reviewer: ```suggestion

To reduce the risk of intrusion of a compromised owner, it is recommended to limit the number of subscription owners.
```

Contractions are not compliant, "it's" and equivalents should be uncontracted to "it is". After fixing it, Grammarly suggested something and I got struck by inspiration

Important: This should not be in description.adoc, but should be in the "how can I fix" section

This rule raises an issue when a custom role has an assignable scope set to a Subscription or a Management Group and allows all actions (``++*++``)

> ¨

Reviewer: You can keep this, it is still interesting to understand the rule
